src/types/evm_env.rs

```diff
@@ -248,6 +301,29 @@ impl BlockEnv {
             .map(|i| i.excess_blob_gas)
     }
 
+    #[getter]
+    fn blob_gasprice(&self) -> Option<u128> {
+        self.0
+            .blob_excess_gas_and_price
+            .clone()
+            .map(|i| i.blob_gasprice)
+    }
+
+    #[setter]
+    fn set_number(&mut self, number: U256) {
```

Comment:
set_block_number?

Reply:
I think number is correct, given it's usually used as `block.number` - this is consistent with the getter

Reply:
ah, i thought this was the big env class, my bad